fetch/src/main/scala/fetch.scala

```diff
@@ -360,6 +361,27 @@ object `package` {
     def pure[F[_]: Applicative, A](a: A): Fetch[F, A] =
       Unfetch(Applicative[F].pure(Done(a)))
 
+    /**
+     * Given a number of fetches, returns all of the results in a `List`. In the event that multiple
+     * fetches are made to the same data source, this will attempt to batch them together.
+     *
+     * This should be used in code that previously relied on the auto-batching behavior of calling
+     * `traverse` on lists of `Fetch` values.
+     */
+    def batchAll[F[_]: Monad, A](fetches: Fetch[F, A]*): Fetch[F, List[A]] = {
+      fetches.toNeSeq
+        .map { nes =>
+          nes
+            .map(_.map(Chain.one(_)))
+            .reduceLeft { (fa, fb) =>
+              val mon = Monad[Fetch[F, *]]
```

Comment:
super-nitpick, but is there any reason not to inline this into the line below?

Reply:
Probably just an oversight on my part while I was trying to get things to compile. Metals was being weird at the time.

Reply:
Inlined :+1: